<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>278</width>
    <height>817</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Library browser</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTreeView" name="tree">
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="buttonPreview">
       <property name="text">
        <string>Preview</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="framePreview">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>256</width>
       <height>256</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>256</width>
       <height>256</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="buttonOptions">
       <property name="text">
        <string>Options</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frameOptions">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="checkOnline">
        <property name="toolTip">
         <string>If this is checked, the library doesn't need to be installed. Contents will be fetched online.</string>
        </property>
        <property name="text">
         <string>Online mode</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkWebSearch">
        <property name="toolTip">
         <string>Open the search results inside FreeCAD's web browser instead of the system browser</string>
        </property>
        <property name="text">
         <string>Open search in FreeCAD web view</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="check3DPreview">
        <property name="toolTip">
         <string>Opens a 3D preview of the selected file.</string>
        </property>
        <property name="text">
         <string>Preview model in 3D view</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkFCStdOnly">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>If the FCStd format is available for a certain file, do not display other file formats (STEP, IFC, etc...)</string>
        </property>
        <property name="text">
         <string>Display only FCStd if available</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonInsert">
     <item>
      <widget class="QPushButton" name="buttonLink">
       <property name="text">
        <string>Link</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="buttonSave">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Note: STEP and BREP files can be placed at custom location. FCStd and IFC files will be placed where objects are defined in the file.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Search:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="searchBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="buttonBimObject">
       <property name="minimumSize">
        <size>
         <width>16</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Search BimObject website</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonNBSLibrary">
       <property name="minimumSize">
        <size>
         <width>16</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Search National Building Library website</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonBimTool">
       <property name="minimumSize">
        <size>
         <width>16</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Search BimTool website</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button3DFindIt">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>16</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Search 3DFindIt website</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonGrabCad">
       <property name="minimumSize">
        <size>
         <width>16</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
